packages/app/src/files-manager/components/unsupported-file-format/unsupported-file-format.component.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import {
+  ModalContent,
+  ModalDialog,
+  RoundIconWrapper,
+} from "App/ui/components/modal-dialog"
+import { ModalSize } from "App/__deprecated__/renderer/components/core/modal/modal.interface"
+import { intl } from "App/__deprecated__/renderer/utils/intl"
+import { defineMessages } from "react-intl"
+import Icon from "App/__deprecated__/renderer/components/core/icon/icon.component"
+
+import React from "react"
+import { IconType } from "App/__deprecated__/renderer/components/core/icon/icon-type"
+import Text, {
+  TextDisplayStyle,
+} from "App/__deprecated__/renderer/components/core/text/text.component"
+import styled from "styled-components"
+import { FunctionComponent } from "App/__deprecated__/renderer/types/function-component.interface"
+import { useSelector, useDispatch } from "react-redux"
+import { resetUploadingState } from "App/files-manager/actions/base.action"
+import { getFilesManagerError } from "App/files-manager/selectors/get-files-manager-error.selector"
+import { FilesManagerError } from "App/files-manager/constants/errors.enum"
+
+const messages = defineMessages({
+  unsupportedFileFormatUploadModalTitle: {
+    id: "module.filesManager.unsupportedFileFormatUploadModalTitle",
+  },
+  unsupportedFileFormatUploadModalHeader: {
+    id: "module.filesManager.unsupportedFileFormatUploadModalHeader",
+  },
+  unsupportedFileFormatUploadModalTextInfo: {
+    id: "module.filesManager.unsupportedFileFormatUploadModalTextInfo",
+  },
+  unsupportedFileFormatUploadModalActionButton: {
+    id: "module.filesManager.unsupportedFileFormatUploadModalActionButton",
+  },
+})
+
+const DuplicatedFilesDetailText = styled(Text)`
+  margin-top: 0.8rem;
+`
+
+const UnsupportedFileFormat: FunctionComponent = ({ ...props }) => {
```

Comment:
Let's maybe use word "modal" in this component's name as in every other component using ModalDialog.